lib/osf-components/addon/components/resources-list/resource-card/template.hbs

```diff
@@ -0,0 +1,16 @@
+<li local-class='resourceItem'>
+    <p local-class='resourceType'>
+        <span local-class='icon {{this.resourceIconClass}}' />
```

Comment:
I've opted to use a span here, an element with no semantic meaning, since this badge icon is presentational and made redundant by the text immediately next to it.